Entities/AEmitter.cpp

```diff
@@ -545,32 +546,28 @@ void AEmitter::Update()
                         pParticle->SetLifetime(pParticle->GetLifetime() * throttleFactor);
 
                     // Let particle loose into the world!
-//                    g_MovableMan.AddParticle(pParticle);
-                    // Might be an Actor...
                     g_MovableMan.AddMO(pParticle);
                     pParticle = 0;
                 }
             }
         }
-        m_LastEmitTmr.Reset();
+		m_LastEmitTmr.Reset();
 
         // Apply recoil/push effects. Joint stiffness will take effect when these are transferred to the parent.
         if (!pushImpulses.IsZero()) { AddImpulseForce(pushImpulses); }
 
         // Count the the damage caused by the emissions, and only if we're not bursting
-        if (!m_BurstTriggered)
-            m_DamageCount += (float)emissions * m_EmitDamage * m_EmitterDamageMultiplier;
-        // Count the the damage caused by the burst
-        else
-            m_DamageCount += m_BurstDamage * m_EmitterDamageMultiplier;
+		if (!m_BurstTriggered) {
+			m_DamageCount += (float)emissionCountTotal * m_EmitDamage * m_EmitterDamageMultiplier;
```

Comment:
If you're already redoing this line, please use `static_cast<float>(emissionCountTotal)` here.